htdocs/core/modules/cheque/doc/pdf_blochet.class.php

```diff
@@ -342,7 +342,30 @@ public function Body(&$pdf, $pagenb, $pages, $outputlangs)
 		$num = count($this->lines);
 		for ($j = 0; $j < $num; $j++)
 		{
-		    $lineinpage++;
+            // Dynamic max line heigh calculation
+            $dynamic_line_height = array();
+            $dynamic_line_height[] = $pdf->getStringHeight(60, $outputlangs->convToOutputCharset($this->lines[$j]->bank_chq));
+            $dynamic_line_height[] = $pdf->getStringHeight(80, $outputlangs->convToOutputCharset($this->lines[$j]->bank_chq));
```

Comment:
I think it should be ->emetteur_chq here ?

Reply:
Yes, it's a mistake form me. Sorry.
I see it was fixed before merge.